src/backend/common/helpers/tests/matchstats_helper_test.py

```diff
@@ -32,12 +34,22 @@ def assert_stats_equal(stats: EventMatchStats, expected_stats: EventMatchStats)
         assert team_stats.keys() == expected_stats[stat].keys()
 
 
-def assert_coprs_equal(
+def assert_coprs_keys_equal(
+    coprs: EventComponentOPRs, expected_coprs: EventComponentOPRs
+) -> None:
+    assert coprs.keys() == expected_coprs.keys()
+    for component, oprs in coprs.items():
+        assert oprs.keys() == expected_coprs[component].keys()
+
+
+def assert_coprs_values_equal(
     coprs: EventComponentOPRs, expected_coprs: EventComponentOPRs
 ) -> None:
     assert coprs.keys() == expected_coprs.keys()
     for component, oprs in coprs.items():
         assert oprs.keys() == expected_coprs[component].keys()
+        for team_key, opr in oprs.items():
+            assert opr == approx(expected_coprs[component][team_key])  # pyre-ignore[16]
```

Comment:
This is `# pyre-ignore[16]`'ed, because it's an ongoing lack of annotations on `pytest`'s part:
https://github.com/pytest-dev/pytest/issues/7469#issuecomment-1491644967